src/Host/Client/Impl/DataInspection/REvaluationResultProperties.cs

```diff
@@ -41,23 +41,45 @@ public enum REvaluationResultProperties : ulong {
         FlagsProperty = 1 << 10,
         /// <seealso cref="IRActiveBindingInfo.ComputedValue"/>
         ComputedValueProperty = 1 << 11,
+        /// <seealso cref="IRValueInfo.CanCoerceToDataFrame"/>
+        CanCoerceToDataFrameProperty = 1 << 12,
         /// <seealso cref="IRValueInfo.HasChildren"/>
         HasChildrenProperty = ExpressionProperty | LengthProperty | AttributeCountProperty | SlotCountProperty | NameCountProperty | FlagsProperty,
     }
 
+    internal class REvaluationResultFieldNames {
```

Comment:
Let's make it a nested class inside `REvaluationResult` (can use `partial` to keep it in a separate file), rename it to `FieldNames`, and drop the `FieldName` prefix as redundant. So external clients would do `REvaluationResult.FieldNames.Repr` etc, and inside the class and its children, you can just do `FieldNames.Repr`.